src/ocispec/ocispec.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/fs/fs.h>
+#include <zephyr/sys/printk.h>
+
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "ocispec.hpp"
+
+aos::Error OCISpec::LoadImageSpec(const aos::String& path, aos::oci::ImageSpec& imageSpec)
+{
+    aos::LockGuard lock(mBufferMutex);
+
+    auto redRet = ReadFileContentToBuffer(path, cJsonMaxContentSize);
```

Comment:
why it is `red` and not `green`? :) Do you mean `readRet`?

Reply:
fixed